src/components/map/sectors/MapVatglassesPosition.vue

```diff
@@ -0,0 +1,71 @@
+<script setup lang="ts">
```

Comment:
Add a template here

```suggestion
<template>
    <slot/>
</template>
```

Reply:
For the slot i get this error:
Slots must be explicitly defined.eslint[vue/require-explicit-slots](https://eslint.vuejs.org/rules/require-explicit-slots.html)

Reply:
Roger. You can define it using defineSlots, or add an empty div instead